tests/Integration/Psr18ClientRequestGenerationTest.php

```diff
@@ -172,8 +172,8 @@ public function createdGetRequestsData()
                 'X-Redmine-API-Key: access_token'.\PHP_EOL.
                 'Content-Type: application/octet-stream'.\PHP_EOL.
                 \PHP_EOL.
-                'This is a test file.'.\PHP_EOL.
-                'It will be needed for testing file uploads.'.\PHP_EOL
+                'This is a test file.'."\n".
```

Comment:
why do we need to get rid of PHP_EOL?

Reply:
The test file has `\n` as line endings, but `\PHP_EOL` on windows machines is `\r\n,` so the tests fails on windows.